documentation/asciidoc/computers/compute-module/introduction.adoc

```diff
@@ -224,7 +224,7 @@ The CM5 Cooler helps dissipate heat from your CM5, improving CPU performance, lo
 .CM5 Cooler
 image::images/cm5-cooler.jpg[alt="CM5 Cooler", width="60%"]
 
-To mount the Cooler to your CM5, attach the thermally conductive silicone at the bottom of the Cooler to the top of your CM5. Align the cut-out in the heatsink with the antenna https://en.wikipedia.org/wiki/Hirose_U.FL[U.FL connector]. As the CM5 heats during use, the silicone will gradually attach more securely to the heat sinks on the CM5.
+To mount the Cooler to your CM5, attach the thermally conductive silicone at the bottom of the Cooler to the top of your CM5. Align the cut-out in the heatsink with the antenna https://en.wikipedia.org/wiki/Hirose_U.FL[U.FL connector]. Optionally, fasten screws in the mounting points found in each corner to secure the Cooler. If you omit the screws, the bond between your Cooler and your Compute Module will improve through time, use, and trust.
```

Comment:
Sounds like solid life advice! :laughing:

Reply:
I admittedly got a little cheeky with some of the CM5 accessories doc.